src/Core/Platform/Push/Services/MultiServicePushNotificationService.cs

```diff
@@ -131,30 +132,36 @@ public Task PushSyncSendDeleteAsync(Send send)
         return Task.FromResult(0);
     }
 
-    public Task SendPayloadToUserAsync(string userId, PushType type, object payload, string identifier,
-        string deviceId = null)
+    public Task PushSyncOrganizationStatusAsync(Organization organization)
     {
-        PushToServices((s) => s.SendPayloadToUserAsync(userId, type, payload, identifier, deviceId));
+        PushToServices((s) => s.PushSyncOrganizationStatusAsync(organization));
         return Task.FromResult(0);
     }
 
-    public Task SendPayloadToOrganizationAsync(string orgId, PushType type, object payload, string identifier,
-        string deviceId = null)
+    public Task PushSyncOrganizationCollectionManagementSettingsAsync(Organization organization)
     {
-        PushToServices((s) => s.SendPayloadToOrganizationAsync(orgId, type, payload, identifier, deviceId));
-        return Task.FromResult(0);
+        PushToServices(s => s.PushSyncOrganizationCollectionManagementSettingsAsync(organization));
+        return Task.CompletedTask;
```

Comment:
Thanks for not continuing with doing `Task.FromResult(0)`. I really need to update all of those in the codebase.